agent/lib/src/health.dart

```diff
@@ -166,9 +166,15 @@ Future<HealthCheckResult> closeIosDialog(
     await unlockKeyChain();
     // Runs the single XCUITest in infra-dialog.
     await inDirectory(dialogDir, () async {
-      String command =
-          'xcrun xcodebuild -project infra-dialog.xcodeproj -scheme infra-dialog -destination id=${d.deviceId} test';
-      Process proc = await pm.start(command.split(' '), workingDirectory: dialogDir.path);
+      List<String> command =
+          'xcrun xcodebuild -project infra-dialog.xcodeproj -scheme infra-dialog -destination id=${d.deviceId} test'.split(' ');
+      // Overwrites code signing config when that exists.
```

Comment:
Can you please add more info on why we need to overwrite the code signing config?

Reply:
Added.